### ISSUE TITLE
Guava 31.1 is available

### ISSUE THREAD

**Issue by jeff-huston** [jjohannes/missing-metadata-guava#5]
Hello,

Thank you for this plugin. It has helped us untangle some of these issues even though we don't fully understand how it all works :). Guava 31.1.1 was released on Feb 28, 2022. It looks like this plugin currently supports up to Guava 31.0.1.

We are running into a build error trying to update to Exoplayer 2.17.0, since that library now pulls in Guava 31.1.1.
Do you have plans to add support for Guava 31.1.1 to this plugin soon?

We are using exclude group for guava on our exoplayer dependency, but that doesn't seem to be enough to resolve the issue.
```
implementation "com.google.android.exoplayer:exoplayer:$exo_player_version", {
        exclude group: 'com.google.guava', module: 'guava'
}
```

Thank you again for this helpful plugin!

**Comment by jeff-huston** [comment]
After further review, I misread the error. I'm going to close this issue and dig deeper into the problem. Sorry for the premature report.

**Comment by jjohannes** [comment]
I will reopen the issue, because I am quite sure I need to update the Plugin for Guava 31.1.x. I will do that soon.

Thanks for opening the issue.

**Comment by jjohannes** [comment]
Fixed in release `31.1` (version of the plugin is now aligned with the max. supported Guava version)

**Comment by jeff-huston** [comment]
Thanks for the fast turnaround, and great idea to align the versions!
